codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -400,7 +400,7 @@ protected void decode(ChannelHandlerContext ctx, ByteBuf buffer, List<Object> ou
 
     @Override
     protected void decodeLast(ChannelHandlerContext ctx, ByteBuf in, List<Object> out) throws Exception {
-        decode(ctx, in, out);
+        super.decodeLast(ctx, in, out);
```

Comment:
I guess `decodeLast` ultimately just called `decode` so it just worked ... but perhaps a bit fragile.